stable/sumologic-fluentd/templates/daemonset.yaml

```diff
@@ -117,3 +121,9 @@ spec:
         - name: docker-logs
           hostPath:
             path: /var/lib/docker
+      {{- if .Values.daemonset.tolerations }}
```

Comment:
Nit: the `if` and `else` statements should be indented only one level less than the block they wrap.